python/snewpy/flavor_transformation.py

```diff
@@ -455,6 +455,9 @@ def __init__(self, mix_angles=None, mh=MassHierarchy.NORMAL):
         else:
             pars = MixingParameters(mh)
             theta12, theta13, theta23 = pars.get_mixing_angles()
+            
+        pars = MixingParameters(mh)
+        deltam21_2, deltam31_2, deltam32_2 = pars.get_mass_square_differences()            

```

Comment:
Now the problem is that `pars` just gets generated each time, so it will hold only the default best fit values for the given `mh`.
I suggest just directly pass MixingParameters instance here into flavor transformation constructor